install/install.sh

```diff
@@ -18,15 +18,19 @@ echo ""
 echo "* Updating package list        "
 apt-get update >> "$install_log" 2>&1
 echo "* Installing nginx, python & pip      "
-apt-get -q -y install dnsutils nginx python python-dev python-pip >> "$install_log" 2>&1
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
```

Comment:
I'd rather see a statement that installs the common dependencies, and then a small section to just install platform specific.

I also wonder why it's necessary to install those, while they're seemingly present by default on an rpi...

Reply:
Fixed that in the new commit.

Ummm, I haven't got my rpi yet. But it seems that they are needed under ubuntu.